pyquil/api/_qpu_client.py

```diff
@@ -103,6 +103,7 @@ def __init__(
         self,
         *,
         quantum_processor_id: str,
+        endpoint_id: Optional[str] = None,
```

Comment:
Surprised linting doesn't force us to move kwarg with default after kwarg without default in signature. Easier to read IMO.

Reply:
I don't mind it, I went with alphabetical ordering without re-arranging the other members. The default is ok because it's kwarg-only and order doesn't matter for those. If there's a concrete style guide that says I should move it I can do that

Reply:
I don't know of a concrete style guide, but it does feel weird since it's not allowed in other places (e.g. non-keyword-only functions, dataclasses). As I pointed out elsewhere, I think it's more obvious to readers which parameters are required if they come first.